### DISCUSSION TITLE
How to Get Involved?

### DISCUSSION THREAD

**Discussion by antranapp** [MobileNativeFoundation/discussions#140] [General]
Hi everyone,

I have read through a lot of discussions here and they are all very valuable.

I'm curious though. I see a lot of companies logo in the front page:

<img width="1060" alt="Screenshot 2021-10-12 at 4 15 07 PM" src="https://user-images.githubusercontent.com/478757/136918539-e4f6c30d-4ffa-4c0b-8eb6-de20688f03eb.png">

What does "Supported By" mean?

If I'd like to advocate my company to support this initiative, what should I do? what are requirements and commitments?

It would be great to understand more about this great initiative.

Best,

An

**Comment by struys** [comment]
same question , are there initiatives/working groups where we can actively contribute to building and growing the organization?

**Comment by StarLard** [comment]
Would also love to know the answer to this and potentially get involved!

**Comment by ob** [comment]
@antranapp Thanks for starting this conversation. Right now "supported by" only means that individuals from that company are actively participating in the community. AFAIK the MNF has not formalized any specific ways to get involved other than participating in these forums. We have bounced around some ideas with @Keith and other folks about what other ways we could provide to get more people actively involved.

One thought is that you could join the [MNF public Slack](https://mobilenativefdn.slack.com).

**Reply by scottluxenberg** [reply]
is there a proper invite link to actually join the slack?

**Reply by bootstraponline** [reply]
I can send you an invite. Let me know what email you'd like to use.

**Reply by scottluxenberg** [reply]
Can you send it to scott.luxenberg@gmail.com?

**Reply by bootstraponline** [reply]
Done!

**Reply by serjooo** [reply]
May I please get an invite as well at serjagopian@gmail.com ? Thank you!

**Reply by sanarroya** [reply]
Can I get an invite as well at sanarroya@gmail.com? Thanks!

**Reply by bootstraponline** [reply]
Invited!

**Reply by piannaf** [reply]
@bootstraponline I'd love an invite to the Slack group, too. justin.mancinelli@gmail.com

I checked with @keith back in June and he said discussion should happen in github. https://github.com/MobileNativeFoundation/discussions/discussions/28#discussioncomment-846428

Has there been an official change of heart or just more options. And are there rules around what discussions should be in Slack vs in Github?

**Reply by bootstraponline** [reply]
Keith is correct. We're encouraging discussions to be on GitHub wherever possible so that there's visibility and everyone can contribute.

Slack has been useful for strategic partnerships between companies to advance the mission of developing at scale.

**Reply by piannaf** [reply]
> Slack has been useful for strategic partnerships between companies to advance the mission of developing at scale.

That makes a lot of sense, and what I was hoping to hear. I think github discussions are great to keep community topics organized, searchable, and threaded.

Thanks for the slack invite!

**Reply by antranapp** [reply]
@bootstraponline is it possible to send you my email privately? Don't want to publish my email here, probably get a lot of spam immediately

**Reply by bootstraponline** [reply]
Sure, you can email me at mnf@bootstraponline.com
